sbin/build.sh

```diff
@@ -104,28 +104,23 @@ configureReproducibleBuildParameter() {
           addConfigureArg "--with-source-date=version"  " --disable-ccache"
           CONFIGURE_ARGS="${CONFIGURE_ARGS//--enable-ccache/}"
       else
-          if [[ -n "${BUILD_CONFIG[BUILD_REPRODUCIBLE_DATE]}" ]]; then
-              # Use supplied date
-              addConfigureArg "--with-source-date=" "${BUILD_CONFIG[BUILD_REPRODUCIBLE_DATE]}"
-
-              # Specify --with-hotspot-build-time to ensure dual pass builds like MacOS use same time
-              # Use supplied date
-              addConfigureArg "--with-hotspot-build-time=" "${BUILD_CONFIG[BUILD_REPRODUCIBLE_DATE]}"
+          # Use BUILD_TIMESTAMP date
```

Comment:
just thinking, since we will not build jdk18 any more, e.g line 96 can be simplify to` if [[ "${BUILD_CONFIG[OPENJDK_FEATURE_NUMBER]}" -ge 17 ]]`

Reply:
I think i'd rather leave it, as these scripts are for any user to use, and they may want for some reason to build 18